app/models/spree/calculator/weight/package_builder.rb

```diff
@@ -0,0 +1,134 @@
+module Spree
+  module Calculator::Weight
```

Comment:
This isn't a Calculator, so it would be nice to pull it out of the Calculator namespace.

Maybe something like `SolidusActiveShipping::PackageBuilder` to avoid any possible confusion or conflicts.